bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/firmware/FirmwareUpdateService.java

```diff
@@ -414,7 +414,7 @@ private void createFirmwareUpdateStatusInfoJob() {
             logger.debug("Creating firmware status info job. [delay:{}, period:{}, time unit: {}]",
                     firmwareStatusInfoJobDelay, firmwareStatusInfoJobPeriod, firmwareStatusInfoJobTimeUnit);
 
-            firmwareStatusInfoJob = getPool().scheduleAtFixedRate(firmwareStatusRunnable, firmwareStatusInfoJobDelay,
+            firmwareStatusInfoJob = getPool().scheduleWithFixedDelay(firmwareStatusRunnable, firmwareStatusInfoJobDelay,
```

Comment:
@tomhoefer can you give a hint if `scheduleAtFixedRate` is required here?

Reply:
schedule with fixed delay is also fine in this context